crates/bws/src/main.rs

```diff
@@ -603,6 +628,108 @@ async fn process_commands() -> Result<()> {
             }
         }
 
+        Commands::Run {
+            command,
+            shell,
+            no_inherit_env,
+            project_id,
+        } => {
+            let shell = match std::env::consts::OS {
+                os if os == "linux" || os == "macos" || os.contains("bsd") => {
+                    shell.unwrap_or_else(|| "sh".to_string())
+                }
+                "windows" => shell.unwrap_or_else(|| "powershell".to_string()),
+                _ => unreachable!(),
+            };
+
+            if which(&shell).is_err() {
+                eprintln!("Error: shell '{}' not found", shell);
+                std::process::exit(1);
+            }
+
+            let user_command = if command.is_empty() {
+                if atty::is(Stream::Stdin) {
```

Comment:
I intended for this to only work as a non-interactive command, so something like `echo 'output from some other script or input' | bws run` _should_ work, but it should print the help if you try to run it interactively, in the examples that you provided.

I did this because it felt like letting someone pop an interactive shell with secrets in it could encourage risky behavior, like using `bws` as a shell itself, outside of the intended use-cases.